deploy/clowdapp.yaml

```diff
@@ -126,7 +126,7 @@ objects:
     - ingress
     - sources
     database:
-      name: catalog_inventory_production
+      name: catalog_inventory
```

Comment:
it's `catalog_inventory` or `catalog-inventory`?

Reply:
does not matter.